src/main/java/picard/sam/SetNmMDAndUqTags.java

```diff
@@ -48,19 +47,19 @@
  * @author Yossi Farjoun
  */
 @CommandLineProgramProperties(
-        usage = SetNmAndUqTags.USAGE_SUMMARY + SetNmAndUqTags.USAGE_DETAILS,
-        usageShort = SetNmAndUqTags.USAGE_SUMMARY,
+        usage = SetNmMDAndUqTags.USAGE_SUMMARY + SetNmMDAndUqTags.USAGE_DETAILS,
+        usageShort = SetNmMDAndUqTags.USAGE_SUMMARY,
         programGroup = SamOrBam.class
 )
-public class SetNmAndUqTags extends CommandLineProgram {
-    static final String USAGE_SUMMARY = "Fixes the UQ and NM tags in a SAM file.  ";
-    static final String USAGE_DETAILS = "This tool takes in a SAM or BAM file (sorted by coordinate) and calculates the NM and UQ tags by comparing with the reference."+
+public class SetNmMDAndUqTags extends CommandLineProgram {
```

Comment:
It's going to drive me crazy that it's `Nm`, `Uq` and `MD` in the name.  Can you change to `Md` please?